docs/source/index.rst

```diff
@@ -35,9 +35,11 @@ starfish. Finally, advanced users can examine the :ref:`Data Structures <data st
 :ref:`Help & Reference <help and reference>` sections to learn more details about _starfish_ and its
 object models.
 
-For questions, suggestions, or accolades, `join the #starfish-users channel on the CZI Science Slack
-<https://join-cziscience-slack.herokuapp.com/>`_ and say “hi!” or
-`email the starfish team directly <mailto:starfish@chanzuckerberg.com>`_
+Starfish is moving user support from `the #starfish-users channel on the CZI Science Slack
```

Comment:
Can you drop the "is moving" section and the email link? It should read as we want it to be read in 3-9 months so we don't have to edit it again.

Reply:
K done